doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,291 @@
+===============================================
+2D Flow around NACA0012 at low Reynolds number
+===============================================
+
+Features
+--------
+
+- Solver: ``gls_navier_stokes_2d`` (with Q1-Q1)
+- Transient problem
+- Boundary Layer Mesh - Transfinite Mesh
+- Spectral analysis - Fourier transform
+
+Files used in this example
+----------------------------
+
+- Base case (:math:`Re = 1000`): ``/examples/incompressible-flow/2d-naca0012-low-reynolds/naca.prm``
+- .geo file to generate the mesh: ``/examples/incompressible-flow/2d-naca0012-low-reynolds/c_type_mesh.geo``
+
+
+Description of the case
+-------------------------
+
+In this example, a two-dimensional flow around a NACA airfoil is studied. According to Wikipedia, the *NACA airfoils are airfoil shapes for aircraft wings developed by the National Advisory Committee for Aeronautics (NACA)*. The NACA is the ancestor of the well known `NASA <https://www.nasa.gov/about/index.html>`_. The airfoils, simply referred to as NACAs afterwards, can be described by a set of four digits for the simpler airfoils, allowing to generate a lot of different profiles. Since we deal with deal with a symmetrical NACA we will only consider the last two digits which describe the maximum thickness of the airfoil in percentage of the chord. The first two digits, describing the maximum camber and its position, are set to zero. The problem under study is presented in the figure below, which gives details about the boundary conditions chosen and gives an idea of the size of the box for the resolution.
```

Comment:
```suggestion
In this example, a two-dimensional flow around a NACA airfoil is studied. According to Wikipedia, the *NACA airfoils are airfoil shapes for aircraft wings developed by the National Advisory Committee for Aeronautics (NACA)*. The NACA is the ancestor of the well-known `NASA <https://www.nasa.gov/about/index.html>`_. The airfoils, simply referred to as NACAs afterwards, can be described by a set of four digits for the simpler airfoils, allowing the generation of a lot of different profiles. Since we deal with a symmetrical NACA we will only consider the last two digits, which describe the maximum thickness of the airfoil in percentage of the chord. The first two digits, describing the maximum camber and its position, are set to zero. The problem under study is presented in the figure below, which details the boundary conditions chosen and gives an idea of the box size for the resolution.
```